### PR TITLE
fix(DB/spell_proc_event) Add ICD to Spellsurge and some PVP trinkets

### PR DESCRIPTION
**Pull request by v-mstrs**
## Changes Proposed:
- Adds 17s cd to Mark of Defiance
- Adds 50s cd to Spellsurge
- Adds 25s cd to Mark of Vindication and Mark of Conquest
- Adds 4m cd to Primal Blessing Trigger

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/5423,
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/16234
- Closes https://github.com/chromiecraft/chromiecraft/issues/5780
## Tests Performed:

- Seems to work

## How to Test the Changes:

(Alliance) check how often they proc
1. .additem 27921
2. .additem 38926
3. .additem 27922
4. .additem 27926
5. .additem 19896   .additem 19910

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by heyitsbench** [comment]
Spellsurge at least has an ICD around 50 seconds. Can't speak for the others.

**Comment by elthehablo** [comment]
Others based on TC, I presume?

**Comment by heyitsbench** [comment]
> Others based on TC, I presume?

Spellsurge based on Classic testing, Mark of Defiance/Vindication based on Classic Wowhead it seems, Mark of Conquest based on Wowhead comments, and Primal Blessing based on Wowhead comments and a Reddit thread discussing Classic.

**Comment by heyitsbench** [comment]
Build failing on line 7 (should be a comma instead of a semicolon).

File: data/sql/updates/pending_db_world/rev_1686876002032070899.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+--
+DELETE FROM `spell_proc_event` WHERE `entry` IN (27997, 33511, 33522, 33510, 24256);
+INSERT INTO `spell_proc_event` VALUES
+(27997,0,0,0,0,0,0,0,0,0,0,50000),
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1686876002032070899.sql:4]: Always add the field names in the INSERT, don't leave it empty

**Review by Gultask — APPROVED** [review]

**Comment by Gultask** [comment]
33510 already had a row that made it 5 ppm, which made the cooldown way lower than 25s lol

Anyway, thanks for the PR

**Pull request merged**